pkgs/by-name/st/steel-language-server/package.nix

```
@@ -0,0 +1,62 @@
+{
+  lib,
+  rustPlatform,
+  fetchFromGitHub,
+  makeBinaryWrapper,
+  steel,
+
+  nix-update-script,
+}:
+rustPlatform.buildRustPackage {
+  pname = "steel-language-server";
+  version = "0-unstable-2025-10-14";
+
+  src = fetchFromGitHub {
+    owner = "mattwparas";
+    repo = "steel";
+    rev = "c83d48f657f28017912cc3c1b604c33ab5de0a41";
+    hash = "sha256-VVrMXpJwcaUTSR6AElkc7Xu8If0/bUzKklOyo87CdLI=";
+  };
+
+  cargoHash = "sha256-CrmQhOfh7SQ5GvBywmYkfU6wMlgZq2x61+T+mIeQ7z4=";
```

So this will automatically be updated when steel is

```suggestion
  inherit (steel)
    cargoHash
    postPatch
    src
    version
    ;
```